antora-playbook.yml

```diff
@@ -24,8 +24,8 @@ site:
         { "title": "Server", "startPage": "home::server.adoc", "components": ["server"] },
         { "title": "Mobile / Edge", "startPage": "home::mobile.adoc", "components": ["couchbase-lite", "sync-gateway", "couchbase-edge-server"] },
         { "title": "Capella", "startPage": "home::cloud.adoc", "components": ["cloud", "columnar"] },
-        { "title": "Cloud-Native", "startPage": "cloud-native-database::index.adoc", "components": ["cloud-native-database"] },
-        { "title": "Autonomous Operator", "components": ["operator"] },
+        { "title": "Cloud-Native", "components": ["cloud-native-database"] },
+        { "title": "Autonomous Operator", "startPage": "operator::overview.adoc", "components": ["operator"] },
```

Comment:
Is "Autonomous Operator" the preferred wording now?
In the antora preview config we've currently got "Kubernetes Operator" pencilled in, I can't remember if this was agreed or a best guess.